bigtable/google/cloud/bigtable/instance.py

```diff
@@ -100,7 +92,7 @@ def __init__(self,
         self.instance_id = instance_id
         self._client = client
         self.display_name = display_name or instance_id
-        self.instance_type = instance_type
+        self.type = instance_type
```

Comment:
By convention, we use `type_` (to avoid confusion with the `type` built-in).

Reply:
Sure, done